src/ophys_etl/workflows/db/schemas.py

```diff
@@ -203,6 +203,36 @@ def _generate_binary_mask(
             ] = True
         return mask.tolist()
 
+    @property
+    def is_valid(self) -> bool:
+        """Return True if no processing flags (motion_border, decrosstalk)
+        flags are set.
+
+        Returns
+        -------
+        is_valid : bool
+            Composite flag that returns True if no processing flags are set.
+            False if any flag is set.
+        """
+        if self.is_in_motion_border or self.is_in_motion_border is None:
+            return False
+        elif self.is_decrosstalk_ghost or self.is_decrosstalk_ghost is None:
```

Comment:
there are 2 scenarios where these decrosstalk flags could be null:
1. decrosstalk hasn't run
2. decrosstalk doesn't need to run since the experiment isn't mesoscope

Fo 1, I think we should return null here since we don't know if it's valid yet
For 2, we shouldn't check the decrosstalk flags since they don't apply